ui/components/AutomationDetail.tsx

```diff
@@ -49,10 +49,14 @@ function AutomationDetail({ automation, className, info, customTabs }: Props) {
 
   const suspend = useToggleSuspend(
     {
-      name: automation?.name,
-      namespace: automation?.namespace,
-      clusterName: automation?.clusterName,
-      kind: automation?.kind,
+      objects: [
+        {
+          name: automation?.name,
```

Comment:
I don't think it's related to the current PR, but question marks should be probably used consistently in the `AutomationDetail` component.

In `useSyncFluxObject` and `useToggleSuspend` here we access `automation` fields with optional chaining. In `useGetObject` above them we just write `automation.name` and so on.

@ozamosi should question marks be added to the `useGetObject` args or does `useGetObject` always return a valid object, even if empty, and in this case should the question marks be removed from `useSyncFluxObject` and `useToggleSuspend` args?

Reply:
In the `SourceDetail` the `source` prop is not optional.

Reply:
Question marks should be added, because `AutomationDetail` still deals with responses from `GetKustomization` and `GetHelmRelease`. They can return an error, in which case we'll still render the whole component but with `automation = undefined`, which would be unfortunate.

With `useGetObject`, the return value is always a valid object, so in `SourceDetail` it doesn't need all those question marks. But they're still needed here.